cityofaustin/atd-moped
# disable comment save button if empty

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/10996

## Testing
**URL to test:** 

https://deploy-preview-911--atd-moped-main.netlify.app/

**Steps to test:**
- go to a project comments tab
- the 'save' button should be disabled until you've input text
- delete the text, the 'save' button should be disabled again

---
#### Ship list
